influxql/parser.go

```diff
@@ -1777,13 +1794,21 @@ func (p *Parser) parseSortFields() (SortFields, error) {
 func (p *Parser) parseSortField() (*SortField, error) {
 	field := &SortField{}
 
-	// Next token must be ASC, until other sort orders are supported.
-	tok, _, _ := p.scanIgnoreWhitespace()
-	if tok != ASC {
-		return nil, errors.New("only ORDER BY ASC supported at this time")
+	// Scan for IDENT or STRING clause.
+	tok, pos, lit := p.scanIgnoreWhitespace()
+	if tok != IDENT && tok != STRING {
+		return nil, newParseError(tokstr(tok, lit), []string{"identifier"}, pos)
+	}
+	field.Name = lit
```

Comment:
The `// Scan for IDENT...` line through `field.Name = lit` should be changed to...
```
	// Parse field name
	ident, err := p.parseIdent()
	if err != nil {
		return nil, err
	}
	field.Name = ident
```